crates/uv/src/commands/pip_install.rs

```diff
@@ -392,45 +353,92 @@ pub(crate) async fn pip_install(
         .await?
     };
 
-    let options = OptionsBuilder::new()
-        .resolution_mode(resolution_mode)
-        .prerelease_mode(prerelease_mode)
-        .dependency_mode(dependency_mode)
-        .exclude_newer(exclude_newer)
-        .index_strategy(index_strategy)
-        .build();
-
     // Resolve the requirements.
-    let resolution = match resolve(
-        requirements,
-        constraints,
-        overrides,
-        project,
-        &editables,
-        &hasher,
-        &site_packages,
-        &reinstall,
-        &upgrade,
-        &interpreter,
-        &tags,
-        &markers,
-        &client,
-        &flat_index,
-        &index,
-        &resolve_dispatch,
-        options,
-        printer,
-    )
-    .await
-    {
-        Ok(resolution) => Resolution::from(resolution),
-        Err(Error::Resolve(uv_resolver::ResolveError::NoSolution(err))) => {
-            let report = miette::Report::msg(format!("{err}"))
-                .context("No solution found when resolving dependencies:");
-            eprint!("{report:?}");
-            return Ok(ExitStatus::Failure);
+    let resolution = if let Some(ref root) = uv_lock {
+        let encoded = fs::tokio::read_to_string("uv.lock").await?;
```

Comment:
So this assumes that it's in the current working directory?

Reply:
Yes. It's the simplest thing to do I think. I expect this code to be eventually deleted. (Like, I don't expect `uv pip install` should ever read a `uv.lock` file.)